Packs/GoogleApigee/Integrations/GoogleApigeeEventCollector/GoogleApigeeEventCollector_test.py

```diff
@@ -187,3 +193,48 @@ def test_test_module(requests_mock, mocker):
 def test_is_token_valid(mocker, scenario, token_initiate_time, token_expiration_seconds, current_time, result):
     is_token_valid = Client.is_token_valid(token_initiate_time, token_expiration_seconds, current_time)
     assert is_token_valid == result, f'{scenario} - does not match expected value'
+
+def test_get_token_with_username(mocker):
+    client = mock_client(mocker)
+    spy_http = mocker.spy(Client, '_http_request')
+    access_token, expires_in, refresh_token = client.get_token_request()
+    assert access_token == 'access_token'
+    assert expires_in == 3600
```

Comment:
Why do we need the first three assertions? 
relevant to this test and to the next one.